db/migrate/20230217215025_create_published_forms.rb

```diff
@@ -0,0 +1,10 @@
+class CreatePublishedForms < ActiveRecord::Migration[7.0]
+  def change
+    create_table :published_forms do |t|
+      t.references :form, index: true, foreign_key: true
+      t.json :version
```

Comment:
I think this would be better called something like 'snapshot' or 'data', to me it wasn't clear reading the code what 'version' was.

Reply:
I see your point and open for suggestions. `version` could be misleading as you may expect a numeric  value rather than json. @alphagov/forms-devs  any other suggestions/preference on what to call this column which will store a copy of form and its pages as JSON?

Reply:
`json_form_blob` or something! It's hard to to come up with something fairly generic as it encapsulates quite a lot.

Reply:
`data` or `form_data` might lead you to think that its the form fillers responses. `forms_and_pages_data` blocks us adding other tables into the mix. 

Maybe @SamJamCul suggestion `json_form_blob` is generic enough

Reply:
I've renamed the `versions` column to `json_form_blob`